app/Filament/App/Resources/LocationLevelResource.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace App\Filament\App\Resources;
+
+use App\Filament\App\Resources\LocationLevelResource\Pages;
+use App\Filament\App\Resources\LocationLevelResource\RelationManagers\LocationsRelationManager;
+use App\Filament\Traits\IsLookupListResource;
+use App\Models\SampleFrame\LocationLevel;
+use App\Services\HelperService;
+use Filament\Forms\Components\Hidden;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\TextInput;
+use Filament\Forms\Components\Toggle;
+use Filament\Forms\Form;
+use Filament\Infolists\Components\TextEntry;
+use Filament\Infolists\Infolist;
+use Filament\Navigation\NavigationItem;
+use Filament\Resources\Resource;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Contracts\Support\Htmlable;
+use Illuminate\Support\Str;
+
+class LocationLevelResource extends Resource
+{
+    use IsLookupListResource;
```

Comment:
I don't think this is needed here. This does the following:

- makes the resource *not* scoped to the tenant, and then adds a query filter to get any 'global' items (but we won't have global items here, so this isn't needed)
- sets up a default form (but we customise the form anyway), and sets defaults for the navigation badge/color/items - which are really intended to work with the localisable 'choice list' entries - and we override most of them here anyway. 


So lets simplify and remove the trait entirely from this PR.